sdmetrics/demos.py

```diff
@@ -15,14 +15,19 @@ def load_demo(modality='multi_table'):
         modality (str):
             Data modality to load. It can be multi_table, single_table
             or timeseries.
+        new_metadata (bool):
+            Whether or not to return the new metadata.
 
     Returns:
         tuple:
             Real data, Synthetic data, Metadata.
     """
     demo_path = pathlib.Path(__file__).parent / 'demos' / f'{modality}.pkl'
     with open(demo_path, 'rb') as demo_file:
-        return pickle.load(demo_file)
+        last_index = 3
+        if new_metadata:
+            last_index = 4
```

Comment:
why does the last index change? Can we add a comment or change a parameter name to indicate what this is for? Also minor but this can be written on one line as 
```python
last_index = 4 if new_metadata else 3
```

Reply:
The new metadata is stored as the 4th index, so depending on if we want to return it or not, the `last_index` changes. Sounds good, I'll add a comment